src/frontend/src/styles/main.css

```diff
@@ -313,6 +319,12 @@
   --rs-line--thick: calc(var(--rs-line) * 4);
 
   --rs-footer-height: 4rem;
+
+  --rs-landing-page-padding--mobile: calc(var(--vs-stack) * 3);
```

Comment:
these are also too specific I think. Maybe calling it something like split layout would allow for some reuse?

Reply:
Good idea, done!